### PR TITLE
Publish ECS docs for beta.5 (#11267)

### PR DESCRIPTION
**Pull request by usha-mandya**
Publish the latest docs for ECS beta 5 release

### CODE REVIEW

File: engine/context/ecs-integration.md [hunk 1]
```diff
@@ -39,39 +39,50 @@ To deploy Docker containers on ECS, you must meet the following requirements:
   >
   > From the Docker Desktop menu, click **Settings** (Preferences on macOS) > **Command Line** and then turn on the **Enable experimental features** toggle. Click **Apply & Restart** for the changes to take effect.
 
-Check your installation by running the command `docker ecs version`.
-
-Docker not only runs multi-container applications locally, but also enables developers to seamlessly deploy Docker containers on Amazon ECS using a Compose file with the `docker ecs compose up` command. The following sections contain instructions on how to deploy your Compose application on Amazon ECS.
+Docker not only runs multi-container applications locally, but also enables 
+developers to seamlessly deploy Docker containers on Amazon ECS using a 
+Compose file with the `docker  compose up` command. The following sections
```
Comment (StefanScherer) [inline engine/context/ecs-integration.md:44]: There are two spaces, one should be removed.

Comment (StefanScherer) [inline engine/context/ecs-integration.md:44]: There are two spaces, one should be removed.

**Review by StefanScherer — CHANGES_REQUESTED** [review]

**Comment by usha-mandya** [comment]
@StefanScherer PTAL - https://github.com/docker/docker.github.io/pull/11295

**Review by StefanScherer — APPROVED** [review]
LGTM

**Review by StefanScherer — APPROVED** [review]
LGTM

**Pull request merged**